dts/arm/nuvoton/npcx7m7fc.dtsi

```diff
@@ -9,16 +9,22 @@
 
 / {
 	flash0: flash@10070000 {
-		reg = <0x10070000 DT_SIZE_K(320)>;
+		reg = <0x10070000 DT_SIZE_K(256)>;
```

Comment:
Please add a comment that this reallocates the last 64 KiB of code RAM for use as data RAM.

Reply:
done.